neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/view/activity/VariantListActivity.kt

```diff
@@ -85,6 +88,21 @@ class VariantListActivity : AppCompatActivity(), VariantListPresenter.ViewSurfac
         navigate(EditVariantActivity::class, EditVariantPresenter.ViewSurface.EXTRA_NAME to name)
     }
 
+    override fun goToMainApplication() {
+        val resolvedInfos = packageManager.queryIntentActivities(Intent(Intent.ACTION_MAIN), PackageManager.GET_RESOLVED_FILTER)
```

Comment:
@AlgoRyan I initially went with this approach, however on my current project we are overriding the manifest on debug builds to make neanderthal the default launcher, which would mean this would launch itself